### ISSUE TITLE
outlook-matters should use the correct office version

### ISSUE THREAD

**Issue by maxlmo** [makmu/outlook-matters#28]
Right now it depends on your office version, whether you can run the project. To change the office version you have to change the [Project Properties](https://github.com/makmu/outlook-matters/blob/master/OutlookMatters/OutlookMatters.csproj)

**Comment by makmu** [comment]
This is a development-only bug and should only affect users who try to start outlook in a visual studio debug session. Nonetheless, we should try to find a way for developers to debug the add-in with all office versions 2010+.

**Comment by makmu** [comment]
The problematic section in the solution file is the following:
```
<ProjectProperties HostName="Outlook" HostPackage="{29A7B9D7-A7F1-4328-8EF0-6B2D1A56B2C1}" OfficeVersion="15.0" VstxVersion="4.0" ApplicationType="Outlook" Language="cs" TemplatesPath="" DebugInfoExeName="#Software\Microsoft\Office\16.0\Outlook\InstallRoot\Path#outlook.exe" AddItemTemplatesGuid="{A58A78EB-1C92-4DDD-80CF-E8BD872ABFC4}" />
```
where Visual Studio goes directly to the registry to find the installation path of outlook (which btw, I find very strange). In this section of the project file macros and variables cannot be used, unfortunately.

One option here would be to duplicate the respective sections in the registry (of the other office installation like 14.0 or 15.0) and replace the 14.0 or 15.0 by 16.0 in the path. This I find even more strange than asking the registry for the installation path in the first place (to the extend of almost being offensive^^). http://stackoverflow.com/questions/18783931/vsto-tools-office-2010-to-2013-upgrade

So I opt for another solution: We could restructure the repository to only include the ThisAddin Class in the main project. All other files could go into a new assembly, e.g. "Core". Developers with an office version other than 16.0 could then just hand-edit the main project file and replace the "16.0" part by their office version and ignore this change when checking-in. This workaround should be mentioned in the contributing section of the global readme file.

Further References:
http://stackoverflow.com/questions/18250511/cannot-debug-or-run-word-addin-because-the-required-version-of-microsoft-office

**Comment by makmu** [comment]
fixed by restructuring project #40
